must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,20 +191,36 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
+    #Add ocs ready nodes to the list
     nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
 
-    # Collecting ceph prepare volume logs
+    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
+        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
+    done
+
+    oc delete -f pod_helper.yaml
+
     volume_collection(){
+        NODE_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/volume_collection_${node}
+        mkdir -p "${NODE_OUTPUT_DIR}"
         printf "collecting prepare volume logs from node %s \n"  "${node}"
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/log "${NODE_OUTPUT_DIR}"
     }
 
-    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
-        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
-    done
+    journal_collection(){
+        JOURNAL_LOGS=${CEPH_COLLECTION_PATH}/journal_log_${node}
+        mkdir -p "${JOURNAL_LOGS}"
+        printf "collecting journal logs from %s \n" "${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/journal.log "${JOURNAL_LOGS}"
+    }
```

Comment:
move this function clustered_scope file

Reply:
resolving this after discussing it offline.